### PR TITLE
Added Sevenbroker to the list of wallets

### PR DESCRIPTION
**Pull request by MarkSevenB**
Logo added.
Description added.

### CODE REVIEW

File: _data/wallets.yml [hunk 1]
```diff
@@ -1,3 +1,10 @@
+- name: SevenB
```
Comment (jeroz1) [inline _data/wallets.yml:1]: would you mind keeping it in alphabetical order?

**Review by jeroz1 — COMMENTED** [review]
I'd recommend removing the commits with the wrong file location to clean up the history and using a transparent png logo like the rest of the logos, such that any background change will not affect the visibility.

File: _data/wallets.yml (hunk 1 above)
Comment (MarkSevenB) [reply _data/wallets.yml:1]: i don't mind

**Comment by MarkSevenB** [comment]
> I'd recommend removing the commits with the wrong file location to clean up the history and using a transparent png logo like the rest of the logos, such that any background change will not affect the visibility.

okay

**Pull request closed without merging**